week 2/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+<head>
+    <title>Currency Conversion</title>
+    
+</head>
+<body>
+    <div id="timer" class="timer-container"></div>
+    
+    <div class="form-container">
+        <h2>Insert a New Currency Rate</h2>
+        <button class="collapsible">Insert a New Currency Rate</button>
+        <div class="content">
+            <form id="insertCurrencyForm">
+                <label for="base">Base Currency:</label><br>
+                <input type="text" id="base" name="base" value="EUR" disabled><br>
+                <label for="targetCurrency">Target Currency:</label><br>
+                <input type="text" id="targetCurrency" name="targetCurrency"><br>
+                <label for="rate">Rate:</label><br>
+                <input type="number" step="0.00001" id="rate" name="rate"><br><br>
+                <input type="button" value="Insert Rate" onclick="insertCurrencyRate()">
```

Comment:
As mentioned in the previous PRs, don't bind events using HTML attributes; do it with `addEventListener`. I write a single comment on this line but it applies to the rest of this HTML file.

Reply:
Done.